main/wallet.js

```diff
@@ -147,9 +131,14 @@ async function updateScheduledRewards () {
 
 async function _updateScheduledRewards () {
   assert(ctx)
-  scheduledRewards = await backend.fetchScheduledRewards()
```

Comment:
if we don't update `scheduledRewards`, future calls to `getScheduledRewards` will return the outdated value

Reply:
We update it in `main/core.js` line 88 with `wallet.setScheduledRewards`

Reply:
i.e. it is updated from core rather than from the backend

Reply:
Yes but the value in the backend will be wrong afterwards, right?

Reply:
Unless I am wrong, we don't use the value in the backend anywhere, or store it as state in the backend. Global search for `fetchScheduledRewards` just shows the function and its test now so could even delete it.

Reply:
Yeah, but someone could start using it again. If you want to keep it static then please add a code comment that it will get outdated

Reply:
Ok good catch thanks - I now make sure to fetch from the wallet store each time, not from the state in the wallet file.